src/Transactions/BankTransactionLine/Base.php

```diff
@@ -52,6 +56,29 @@
      */
     private $freeChar;
 
+    /**
+     * @var BankTransaction
+     */
+    private $transaction;
+
+    /**
+     * @return BankTransaction
+     */
+    public function getTransaction(): LinesField
+    {
+        return $this->transaction;
+    }
+
+    /**
+     * @param BankTransaction $object
```

Comment:
`$object` is a bit generic.

Also, why is this type hinted as as `BankTransaction` in the `@param` and `@return`, but the code type hints it as `LinesField`.